src/Vimeo/Vimeo.php

```diff
@@ -124,81 +124,53 @@ public function request($url, $params = array(), $method = 'GET', $json_body = t
         }
 
         // Set the headers
-        $curl_opts[CURLOPT_HTTPHEADER] = $headers;
+        $curlOpts[CURLOPT_HTTPHEADER] = $headers;
 
-        $response = $this->_request($curl_url, $curl_opts);
+        $response = $this->doRequest($curlUrl, $curlOpts);
```

Comment:
This will recursively call itself. Believe this should probably call the renamed `request` method instead.